org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/theme/ThemeManager.java

```diff
@@ -79,11 +79,14 @@ public static String getTheme() {
 		String theme = SystemProperties.getZkTheme();
 		if (Util.isEmpty(theme))
 			theme = MSysConfig.getValue(MSysConfig.ZK_THEME, ITheme.ZK_THEME_DEFAULT);
+		if (theme.contains(","))
+			theme = theme.split(",")[0];
 		if (theme.equals(m_brokenTheme)) {
 			theme = ITheme.ZK_THEME_DEFAULT;
 		} else {
 			if (! theme.equals(m_theme)) {
 				if (! ITheme.ZK_THEME_DEFAULT.equals(theme)) {
+					// NOTE: Here we could use also Themes.hasTheme instead, but be careful that default is registered as breeze
 					// Verify the theme.css.dsp exists in the theme folder
 					String themeCSSURL = getStyleSheet(theme);
 					if (ThemeManager.class.getResource(toClassPathResourcePath(themeCSSURL)) == null) {
```

Comment:
to implement cookie base user theme selection, we need to change ThemeManager.getTheme to get from cookie (if available) as first priority. We can't use Theme.setTheme to change iDempiere theme, that wouldn't work.